app/modules/session/buttons.tsx

```diff
@@ -36,6 +36,7 @@ export function SignInWithGoogleButton() {
         data-text="continue_with"
         data-size="large"
         data-logo_alignment="left"
+        data-width="272"
```

Comment:
What was the behavior of the button previous to the change? Did it not expand correctly?

Reply:
I attached the pics above. Before, it was in the center and didn't expand. The idea of the PR was to make all fields equal in width.

With this hard-coded value, it looks a bit better on desktops. Don't know how to make it full-width for mobiles! Created the [issue](https://github.com/social-plan-it/plan-it-social-web/issues/169).